apps/docs/src/basics/typography.md

```diff
@@ -0,0 +1,350 @@
+# Typography
+
+::: danger
+Still content to create!
+:::
+
+The onyx font system includes three different font families that can be used throughout your application, pending on the usecase. It contains several headline styles as well as paragraph styles and mono space styles.
```

Comment:
```suggestion
The onyx font system includes two different font families that can be used throughout your application, pending on the usecase. It contains several headline styles as well as paragraph styles and mono space styles.
```

Reply:
Why two?
Source Sans, Lidl and Kaufland

Reply:
okay, saw it:)